tests/networkpolicy_test.go

```diff
@@ -20,23 +17,16 @@ func assertPingSucceed(ip string, vmi *v1.VirtualMachineInstance) {
 	Expect(err).ToNot(HaveOccurred())
 	defer expecter.Close()
 
-	err = tests.CheckForTextExpecter(vmi, []expect.Batcher{
-		&expect.BSnd{S: fmt.Sprintf("ping -w 3 %s \n", ip)},
-		&expect.BExp{R: "0% packet loss"},
-	}, 60)
-	Expect(err).ToNot(HaveOccurred())
+	Expect(tests.PingFromVMConsole(vmi, ip, "-w 3")).To(Succeed())
 }
 
 func assertPingFail(ip string, vmi *v1.VirtualMachineInstance) {
 	expecter, err := tests.LoggedInCirrosExpecter(vmi)
 	Expect(err).ToNot(HaveOccurred())
 	defer expecter.Close()
 
-	err = tests.CheckForTextExpecter(vmi, []expect.Batcher{
-		&expect.BSnd{S: fmt.Sprintf("ping -w 3 %s \n", ip)},
-		&expect.BExp{R: "100% packet loss"},
-	}, 60)
-	Expect(err).ToNot(HaveOccurred())
+	err = tests.PingFromVMConsole(vmi, ip, "-w 3")
```

Comment:
nit: the `Expect(Something).To(Succeed())` pattern can be used here to: 

`Expect(tests.PingFromVMConsole(vmi, ip, "-w 3")).To(Succeed())`

Reply:
this is outdated, function was removed and changed